src/test/kotlin/com/terraformation/backend/accelerator/db/ApplicationStoreTest.kt

```diff
@@ -315,6 +323,418 @@ class ApplicationStoreTest : DatabaseTest(), RunsAsUser {
+      private lateinit var applicationDeliverableId: DeliverableId
+
+      private lateinit var submissionId1: SubmissionId
+      private lateinit var submissionId2: SubmissionId
+      private lateinit var submissionId3: SubmissionId
+      private lateinit var submissionId4: SubmissionId
+      private lateinit var submissionId5: SubmissionId
+      private lateinit var submissionId6: SubmissionId
+
+      @BeforeEach
+      fun setup() {
+        every { user.canReadProjectDeliverables(any()) } returns true
+        every { user.canReadOrganizationDeliverables(any()) } returns true
+        every { user.canReadModule(any()) } returns true
+        every { user.canReadAllDeliverables() } returns true
+
+        prescreenModuleId =
+            insertModule(
+                name = "Pre-screen",
+                overview = "Pre-screen Overview",
+                phase = CohortPhase.PreScreen,
+            )
+        applicationModuleId =
+            insertModule(
+                name = "Application",
+                overview = "Application Overview",
+                phase = CohortPhase.Application)
+
+        prescreenDeliverableId =
+            insertDeliverable(
+                descriptionHtml = "Pre-screen deliverable description",
+                deliverableCategoryId = DeliverableCategory.Compliance,
+                deliverableTypeId = DeliverableType.Questions,
+                moduleId = prescreenModuleId,
+                name = "Pre-screen deliverable",
+            )
+
+        applicationDeliverableId =
+            insertDeliverable(
+                descriptionHtml = "Application deliverable description",
+                deliverableCategoryId = DeliverableCategory.CarbonEligibility,
+                deliverableTypeId = DeliverableType.Questions,
+                moduleId = applicationModuleId,
+                name = "Application deliverable",
+            )
+
+        // Org 1 Project 1
+        submissionId1 =
+            insertSubmission(
+                deliverableId = prescreenDeliverableId,
+                feedback = "feedback 1",
+                internalComment = "comment 1",
+                projectId = org1ProjectId1,
+                submissionStatus = SubmissionStatus.InReview,
+            )
+
+        submissionId2 =
+            insertSubmission(
+                deliverableId = applicationDeliverableId,
+                feedback = "feedback 2",
+                internalComment = "comment 2",
+                projectId = org1ProjectId1,
+                submissionStatus = SubmissionStatus.InReview,
+            )
+
+        // Org 1 Project 2
+        submissionId3 =
+            insertSubmission(
+                deliverableId = prescreenDeliverableId,
+                feedback = "feedback 3",
+                internalComment = "comment 3",
+                projectId = org1ProjectId2,
+                submissionStatus = SubmissionStatus.InReview,
+            )
+
+        submissionId4 =
```

Comment:
Might be worth removing one of these `insertSubmission()` calls to test that the query treats missing submissions as if they existed with "not submitted" status.